mobile-widgets/qml/MapWidget.qml

```diff
@@ -69,17 +69,10 @@ Item {
 		ParallelAnimation {
 			id: mapAnimation
 			CoordinateAnimation {
-				target: map
-				property: "center"
-				to: map.newCenter
-				duration: 2000
+				target: map; property: "center"; to: map.newCenter; duration: 2000;
```

Comment:
I'll be honest - I don't feel THAT strongly about it. It was mainly a comment when I saw the explicit change... this makes the code more compact
The question is "are these arguments to a function call or are these statements in a procedure". It's somewhere in between, isn't it?
I'm ok to leave them in the more dense format